docs/content/odh/trino/add_more_buckets.md

```diff
@@ -6,7 +6,7 @@
 ## Steps
 
 1. Clone apps repo
-2. Navigate to `apps/kfdefs/overlays/$ENV/$CLUSTER/$TRINO_FOLDER/hive-metastores`
+2. Navigate to `apps/kfdefs/overlays/$ENV/$CLUSTER/trino/hive-metastores`
```

Comment:
This is incorrect, you are looking for `opf-trino`, (see example: https://github.com/operate-first/apps/tree/master/kfdefs/overlays/moc/smaug/opf-trino/hive-metastores)